src/NHibernate/Linq/Visitors/NhPartialEvaluatingExpressionVisitor.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Linq.Expressions;
+using Remotion.Linq.Clauses.Expressions;
+using Remotion.Linq.Parsing;
+using Remotion.Linq.Parsing.ExpressionVisitors;
+using Remotion.Linq.Parsing.ExpressionVisitors.TreeEvaluation;
+
+namespace NHibernate.Linq.Visitors
+{
+	internal class NhPartialEvaluatingExpressionVisitor : RelinqExpressionVisitor
```

Comment:
Does no more implements `IPartialEvaluationExceptionExpressionVisitor`. It was needed for supporting failed attempts at partially evaluating expressions on methods to be translated by the Linq provider. With the `NhEvaluatableExpressionFilter`, we can now avoid those failures by marking methods which should not be evaluated.